tils/jquery-slick.mdx

```diff
@@ -0,0 +1,64 @@
+---
+title: Installing jQuery & Slick
+description: A quick guide on how to setup Slick and jQuery to create carousels/sliders.
+author: Sidharth Subramanian
+date: '2021-08-12'
+category: Slick
+---
+
+## What is Slick?   
```

Comment:
We can skip introductions and definitions in a TIL. Let's keep it specific to your learnings.